cb_api.go

```diff
@@ -27,29 +28,30 @@ type ThreatReport struct {
 }
 
 type ApiInfo struct {
```

Comment:
is it easy to remove these whitespace only changes?

Reply:
No. These were done in a commit by Chris. I can rebase it in though if we really need it.

https://github.com/redcanaryco/cb-event-forwarder/commit/3966b72d198b25f085c2998a3e2ce542ced69eda